### DISCUSSION TITLE
What is the impact on system performance? After installing skywalking backend js

### DISCUSSION THREAD

**Discussion by youksi** [apache/skywalking#10581] [Q&A]
After installing skywalking backend js on my system, I conducted a stress test and found that the throughput decreased by 20%. When the sampling rate is lowered from 50% to 10%, the percentage of throughput decrease seems to be around 20%. I would like to know if the community has specific stress tests or performance data.Thank you！

**Comment by wu-sheng** [comment]
For NodeJS agent, I don't think I have one.  @kezhenxu94

**Reply by kezhenxu94** [reply]
We don't have this test so far

**Reply by youksi** [reply]
> We don't have this test so far

Will there be plans in the future? When installing Skywalking in a system, it is important to have a clear performance impact.

**Reply by wu-sheng** [reply]
This is a community. If someone wants to do, then we will have.
This is not a request-assign work flow ...

**Reply by youksi** [reply]
Got it. Thank you all !
